embedded-voltdb/README.adoc

```diff
@@ -15,6 +15,7 @@
 ==== Consumes (via `bootstrap.properties`)
 
 * `embedded.voltdb.enabled` `(true|false, default is 'true')`
```

Comment:
maybe worth having single common properties section for all containers, so no need to copy paste?

Reply:
what if I want to reuse memsql and do not reuse kafka for example? common property won't help

Reply:
I meant ```embedded.{container}.enabled```